pkg/api/controller_test.go

```diff
@@ -2111,8 +2105,7 @@ func TestAuthorizationWithBasicAuth(t *testing.T) {
 		So(resp.StatusCode(), ShouldEqual, http.StatusForbidden)
 
 		// add create perm on repo
-		conf.AccessControl.Repositories["zot-test"].Policies[0].Actions =
-			append(conf.AccessControl.Repositories["zot-test"].Policies[0].Actions, "create")
+		conf.AccessControl.Repositories["zot-test"].Policies[0].Actions = append(conf.AccessControl.Repositories["zot-test"].Policies[0].Actions, "create") // nolint: lll
```

Comment:
can we get away without the nolint:lll, same other places?

Reply:
lll conflicts with gofumpt. The form I've used is the one suggested by gofumpt.

Reply:
add that part of the comment at the end of the nolint like so:
https://github.com/project-zot/zot/blob/main/pkg/api/routes.go#L369

Reply:
added comments regarding conflicts with gofumpt to all occurrences of nolint: lll